src/Client/Services/ServiceFactory.php

```diff
@@ -61,7 +61,7 @@ public function __construct($apiKey, Context $context, Host $host = null, $apiVe
             $this->host = $host;
         }
         if ($apiVersion == null) {
-            $this->apiVersion = '5.0';
+            $this->apiVersion = '5.3';
```

Comment:
@mstralka Tested this and unfortunately nameapi do not have a /5.3/ endpoint and the requests return 404.
You should revert it back to "5.0"